### ISSUE TITLE
$modifiedSince parameter seems to be ignored

### ISSUE THREAD

**Issue by ArthurHoaro** [alexdebril/feed-io#60]
How to reproduce:

```php
$readFeed = $reader->readSince($feedUrl, new \DateTime());
$newItems = $readFeed->getFeed();
foreach($newItems as $value) {
  var_dump($value->getTitle());
}
```

Result:

All items are returned.

Expected result:

I should only get new items (here, none since I used the current time).

----

I investigated a bit, and it seems that the ModifiedSince filter `init()` method is called before `$feed` is initiated, so `$this->date` is always null. Or I might have totally misunderstand how to use this. 😄

**Comment by alexdebril** [comment]
Hi, 

you're absolutely right and I can't figure out how I could miss this bug. Give me a little bit of time and I'll fix this soon.

Thank you very much for your feedback !

Alex

**Comment by alexdebril** [comment]
This bug is fixed in version 2.4.5

**Comment by ArthurHoaro** [comment]
Thanks for the quick fix.
